SBBCoolHelper.user.js

```diff
@@ -721,11 +721,7 @@ div.disabled {
         }
     }
 
-    function VerifyUUID(uuid) {
-        return /[A-Z0-9]{64}/i.test(uuid);
-    }
+    const VerifyUUID = (uuid) => /^[a-f0-9]{64,65}$/.test(uuid);
```

Comment:
Segments used to be prefixed with the version number + the sha256 hash

1 + 64